float64 R_i
uint16 z_i
float64 y_i_estimate
float64 P_i_estimate
---
float64 y_i 